UPtables/lib/asp/constraints/hard/double_lectures.lp

```diff
@@ -5,7 +5,7 @@
 %%% are grouped if they are adjacent and in the same room. Each
 %%% non-grouped lecture counts as 1 violation.
 
-:- course_component(C,_,_,_,1), d(D), 2 {  assigned(C,D,PE) },
```

Comment:
@felixKubicek what's the point in this condition? After I deleted it, the test was green. So now ```double_lecture``` means actually, that all dates have to be adjacent (are scheduled in a row on the same day).